### DISCUSSION TITLE
Web Support Still Experimental?

### DISCUSSION THREAD

**Discussion by RedHappyLlama** [flutter-stripe/flutter_stripe#732] [Q&A]
Hi,

Is support of web still highly experimental? I only need to be able to take one-off card payments reliably. Is this likely to be an issue in production? 

Thanks!

**Comment by remonh87** [comment]
We are a bit stretched at the moment with the amount of improvements are coming from the Stripe sdk on mobile. This is a blessing but also comes at a cost and that is the web at the moment. Feel free to let us know what you are missing.  I think the functions are there are working but it can be that we are forced to introduce some breaking changes on the web thats why it is marked as experimental
